src/SprykerEco/Zed/Unzer/Dependency/UnzerToCalculationFacadeBridge.php

```diff
@@ -29,7 +29,7 @@ public function __construct($calculationFacade)
      *
      * @return \Generated\Shared\Transfer\OrderTransfer
      */
-    public function recalculateOrder(OrderTransfer $orderTransfer)
+    public function recalculateOrder(OrderTransfer $orderTransfer): OrderTransfer
```

Comment:
Please, revert this change and other similar changes in old bridge classes

Reply:
Fixed